examples/ex10b/MASTODON_Explicit_HEX20.i

```diff
@@ -0,0 +1,250 @@
+[Mesh]
```

Comment:
Same comments as above. Please remove extra empty lines and empty blocks. 
implicit = false is only needed in the strain block. Please remove it from other blocks.